### ISSUE TITLE
NeurIPS, ICLR 2024 Papers

### ISSUE THREAD

**Issue by chufanchen** [chufanchen/read-paper-and-code#190]
## RL

- [A Study of Plasticity Loss in On-Policy Deep Reinforcement Learning](https://arxiv.org/abs/2405.19153)
- [Goal Reduction with Loop-Removal Accelerates RL and Models Human Brain Activity in Goal-Directed Learning](https://github.com/chenghuzi/goal-reducer)
- [The Value of Reward Lookahead in Reinforcement Learning](https://arxiv.org/abs/2403.11637)
- [Reinforcement Learning Gradients as Vitamin for Online Finetuning Decision Transformers](https://arxiv.org/abs/2410.24108)
- [Optimizing Automatic Differentiation with Deep Reinforcement Learning](https://arxiv.org/abs/2406.05027)
- [Personalizing Reinforcement Learning from Human Feedback with Variational Preference Learning](https://arxiv.org/abs/2408.10075)
- [NeoRL: Efficient Exploration for Nonepisodic RL](https://arxiv.org/abs/2406.01175)
- [Sample Complexity Reduction via Policy Difference Estimation in Tabular Reinforcement Learning](https://arxiv.org/abs/2406.06856)
- [Logarithmic Smoothing for Pessimistic Off-Policy Evaluation, Selection and Learning](https://arxiv.org/abs/2405.14335)
- [Rethinking Exploration in Reinforcement Learning with Effective Metric-Based Exploration Bonus](https://github.com/YimingWangMingle/EME)
- [Bigger, Regularized, Optimistic: scaling for compute and sample-efficient continuous control](https://arxiv.org/abs/2405.16158)
- [DiffTORI: Differentiable Trajectory Optimization for Deep Reinforcement and Imitation Learning](https://arxiv.org/abs/2402.05421)
- [Exclusively Penalized Q-learning for Offline Reinforcement Learning](https://arxiv.org/abs/2405.14082)
- [Diffusion for World Modeling: Visual Details Matter in Atari](https://arxiv.org/abs/2405.12399)
- [The Power of Resets in Online Reinforcement Learning](https://arxiv.org/abs/2404.15417)
- [Privileged Sensing Scaffolds Reinforcement Learning](https://arxiv.org/abs/2405.14853)
- [Harnessing Density Ratios for Online Reinforcement Learning](https://arxiv.org/abs/2401.09681)
- [Addressing Signal Delay in Deep Reinforcement Learning](https://openreview.net/forum?id=Z8UfDs4J46)
- [Pre-Training Goal-based Models for Sample-Efficient Reinforcement Learning](https://github.com/PKU-RL/PTGM)
- [METRA: Scalable Unsupervised RL with Metric-Aware Abstraction](https://arxiv.org/abs/2310.08887) 
- [Predictive auxiliary objectives in deep RL mimic learning in the brain](https://arxiv.org/abs/2310.06089)
- [B-Coder: Value-Based Deep Reinforcement Learning for Program Synthesis](https://openreview.net/forum?id=QwNLVId9Df)
- [Massively Scalable Inverse Reinforcement Learning in Google Maps](https://arxiv.org/abs/2305.11290)
- [DrM: Mastering Visual Reinforcement Learning through Dormant Ratio Minimization](https://arxiv.org/abs/2310.19668)

## Accelerating

- [FlashAttention-3: Fast and Accurate Attention with Asynchrony and Low-precision](https://arxiv.org/abs/2407.08608)
- [Accelerating Diffusion Models with Parallel Sampling: Inference at Sub-Linear Time Complexity](https://arxiv.org/abs/2405.15986)
- [MaskLLM: Learnable Semi-Structured Sparsity for Large Language Models](https://arxiv.org/abs/2409.17481)
- [LLMCBench: Benchmarking Large Language Model Compression for Efficient Deployment](https://arxiv.org/abs/2410.21352)
- [MInference 1.0: Accelerating Pre-filling for Long-Context LLMs via Dynamic Sparse Attention](https://arxiv.org/abs/2407.02490)
- [Model Tells You What to Discard: Adaptive KV Cache Compression for LLMs](https://arxiv.org/abs/2310.01801)
- [Scaling Laws for Associative Memories](https://arxiv.org/abs/2310.02984)
- [CO2: Efficient Distributed Training with Full Communication-Computation Overlap](https://arxiv.org/abs/2401.16265)

**Comment by chufanchen** [comment]
## RLHF

- [Universal Jailbreak Backdoors from Poisoned Human Feedback]
- [The Trickle-down Impact of Reward Inconsistency on RLHF]
- [Understanding Hidden Context in Preference Learning: Consequences for RLHF]
- [Contrastive Preference Learning: Learning from Human Feedback without Reinforcement Learning]
- [CPPO: Continual Learning for Reinforcement Learning with Human Feedback]
- [AlignDiff: Aligning Diverse Human Preferences via Behavior-Customisable Diffusion Model]

## Pre-train RL

- [Pre-training with Synthetic Data Helps Offline Reinforcement Learning]()
- [Transformers as Decision Makers: Provable In-Context Reinforcement Learning via Supervised Pretraining]()
- [Pre-Training Goal-based Models for Sample-Efficient Reinforcement Learning](url)

## Agent

- [Retroformer: Retrospective Large Language Agents with Policy Gradient Optimization](https://arxiv.org/abs/2308.02151)
- [Plan-Seq-Learn: Language Model Guided RL for Solving Long Horizon Robotics Tasks]
- [ArCHer: Training Language Model Agents via Hierarchical Multi-Turn RL]
- [FuRL: Visual-Language Models as Fuzzy Rewards for Reinforcement Learning]
- [MAGDi: Structured Distillation of Multi-Agent Interaction Graphs Improves Reasoning in Smaller Language Models]
- [LLM-Empowered State Representation for Reinforcement Learning]()
- [Learning Reward for Robot Skills Using Large Language Models via Self-Alignment]()
- [Language-guided Skill Learning with Temporal Variational Inference]
- [Language Agent Tree Search Unifies Reasoning, Acting, and Planning in Language Models]
- [Language Agents with Reinforcement Learning for Strategic Play in the Werewolf Game]
- [FuRL: Visual-Language Models as Fuzzy Rewards for Reinforcement Learning]
- [Feedback Loops With Language Models Drive In-Context Reward Hacking]
- [From Words to Actions: Unveiling the Theoretical Underpinnings of LLM-Driven Autonomous Systems]
- [Code as Reward: Empowering Reinforcement Learning with VLMs]
- [KALM: Knowledgeable Agents by Offline Reinforcement Learning from Large Language Model Rollouts]


## RL for LM

- [Self-Play Fine-tuning of Diffusion Models for Text-to-image Generation]
- [Fine-Tuning Large Vision-Language Models as Decision-Making Agents via Reinforcement Learning]
- [RL-GPT: Integrating Reinforcement Learning and Code-as-policy]
- [Reinforcing LLM Agents via Policy Optimization with Action Decomposition]
- [Coevolving with the Other You: Fine-Tuning LLM with Sequential Cooperative Multi-Agent Reinforcement Learning]
